<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameImageClass</class>
 <widget class="QMainWindow" name="FrameImageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrameImage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelImagePath">
        <property name="text">
         <string>ImagePath</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_InPutImage"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_OpenImage">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_ShapePath">
        <property name="text">
         <string>ShapeFilePath</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_OutShp"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ShapeFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_OutPath">
        <property name="text">
         <string>OutPath</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_OutImage"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_OutPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Extension(cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_Scale">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxScale">
        <property name="currentIndex">
         <number>9</number>
        </property>
        <item>
         <property name="text">
          <string>1:1000000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:500000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:250000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:100000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:50000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:25000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:10000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:5000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:2000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:500</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_Tools">
        <property name="text">
         <string>Tool</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQualityCheck">
        <property name="text">
         <string>QualityCheck</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Imageshow</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Overview</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Attribute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QListWidget" name="listWidgetFrameList"/>
    </item>
    <item row="5" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFrame">
        <property name="text">
         <string>Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSubset">
        <property name="text">
         <string>Subset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FrameImage.qrc"/>
 </resources>
 <connections/>
</ui>
